subsys/bluetooth/host/Kconfig

```diff
@@ -101,7 +101,8 @@ config BT_RX_STACK_SIZE
 	depends on BT_HCI_HOST || BT_RECV_IS_RX_THREAD
 	default 512 if BT_HCI_RAW
 	default 2048 if BT_MESH
-	default 2048 if BT_SETTINGS
+	default 2048 if (BT_SETTINGS && !BT_SETTINGS_STORE_CCC_IMMEDIATELY)
+	default 2200 if BT_SETTINGS_STORE_CCC_IMMEDIATELY
```

Comment:
Out of curiosity, how did you come up with the 2200 number? Was this a measured value?

Reply:
The lowest value that does not cause the stack guard to be hit (stack overflow).